drivers/pci/probe.c

```diff
@@ -348,6 +348,57 @@ static void pci_read_bases(struct pci_dev *dev, unsigned int howmany, int rom)
 	}
 }
 
+static void pci_read_bridge_windows(struct pci_dev *bridge)
```

Comment:
Check the modem on EDO, if Qualcomm had fantasy about the hypervisor restrictions, this may KO the platform...